ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/polygon/DrawAreaTaskFragment.kt

```diff
@@ -58,16 +58,21 @@ class DrawAreaTaskFragment @Inject constructor() : AbstractTaskFragment<DrawArea
   override fun onCreateTaskBody(inflater: LayoutInflater): View {
     // NOTE(#2493): Multiplying by a random prime to allow for some mathematical "uniqueness".
     // Otherwise, the sequentially generated ID might conflict with an ID produced by Google Maps.
-    val rowLayout = LinearLayout(requireContext()).apply { id = View.generateViewId() * 11411 }
+    val rootView = FragmentDrawAreaTaskBinding.inflate(inflater)
```

Comment:
Please replace the above comment/note with:
```kt
// XML layout is used to provide a static view ID which does not collide with Google Maps view ID (https://github.com/google/ground-android/issues/2493). 
// The ID is needed when restoring the view on config change since the view is dynamically created.
// TODO(https://github.com/google/ground-android/issues/1795):
//   Remove this workaround once this UI is migrated to Compose.
```

Reply:
done